src/commands/account/profile_management/update_profile/profile_args_type/mod.rs

```diff
@@ -0,0 +1,41 @@
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+mod base64_args;
+mod file_args;
+mod json_args;
+mod manually;
+mod text_args;
+
+#[derive(Debug, EnumDiscriminants, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = super::UpdateAccountProfileContext)]
+#[strum_discriminants(derive(EnumMessage, EnumIter))]
+/// How do you want to pass profile arguments?
+pub enum ProfileArgsType {
+    #[strum_discriminants(strum(
+        message = "json-args    - Valid JSON arguments (e.g. {\"token_id\": \"42\"})"
+    ))]
+    /// Valid JSON arguments (e.g. {"token_id": "42"})
+    JsonArgs(self::json_args::JsonArgs),
+    #[strum_discriminants(strum(message = "text-args    - Arbitrary text arguments"))]
+    /// Arbitrary text arguments
+    TextArgs(self::text_args::TextArgs),
```

Comment:
Since the contract only supports valid JSON, let's drop this option to avoid unnecessary confusion.

Reply:
Done